frontend/components/site/SideNav/sideNav.js

```diff
@@ -25,17 +26,22 @@ class SideNav extends React.Component {
         <ul className="site-actions">
           <SideNavItem
             href={this.getUrl(siteId, sideNavPaths.SETTINGS)}
-            icon='settings'
+            icon="settings"
             linkText={sideNavPaths.SETTINGS}
           />
           <SideNavItem
+            href={this.getUrl(siteId, sideNavPaths.USERS)}
+            icon="gear"
+            linkText={sideNavPaths.USERS}
+          />
+          <SideNavItem
             href={this.getUrl(siteId, sideNavPaths.PUBLISHED)}
-            icon='media'
+            icon="media"
```

Comment:
why do we need double quotes and not single?

Reply:
That's a linting thing. In JSX, we use double-quotes for attribute values because that is what our preferred config (`eslint-config-airbnb`) is set to use. Either way doesn't matter, but consistency is good so we just go with what the config specifies :)